modules/installation-user-defined-tags-azure.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+// * installing/installing_azure/installing-azure-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-azure-user-defined-tags_{context}"]
+= Configuring the user-defined tags for Azure
+
+In {product-title} you can use the tags for grouping resources and for managing resource access and cost. You can define the tags on the Azure resources in the `install-config.yaml` file only during {product-title} cluster creation. You cannot modify the user-defined tags after cluster creation.
+
+Support for user-defined tags is available only in {product-title} 4.13 as a Technology Preview (TP) and for the resources created in the Azure Public Cloud. User-defined tags are not supported for the {product-title} clusters upgraded to {product-title} 4.13.
+
+User-defined and {product-title} specific tags are applied only to the resources created by the {product-title} installer and its core operators such as Machine api provider azure Operator, Cluster Ingress Operator, Cluster Image Registry Operator.
+
+By default, {product-title} installer attaches the {product-title} tags to the Azure resources. These {product-title} tags are not accessible for the users.
```

Comment:
Do we need so many instances of `{product-title}`?

Reply:
Actually, there are two types of tags. Azure tags and {product-title} tags. Here in that sentence, we are talking about OCP tags hence using `{product-title} tags` explicitly.